tests/libvmi/vmi.go

```diff
@@ -43,9 +43,14 @@ func New(name string, opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return vmi
 }
 
+// VmiRandName returns a random vmi name, with the pattern: DefaultVmiName + "-" + rand.String(5)
+func VmiRandName() string {
```

Comment:
I don't see the value of this function. We are in the vmi.go module, so the "Vmi" prefix is redudent.

Reply:
Updated